README.md

```diff
@@ -89,3 +89,20 @@ The following table lists the last major version of each package which uses rast
 | `types`                   | 3                                      |
 | `vehicle-rental-overlay`  | 1                                      |
 | `zoom-based-markers`      | 1                                      |
+
+## Internationalization
+
+OTP-UI uses `react-intl` from the [`formatjs`](https://github.com/formatjs/formatjs) library for internationalization.
+Both `react-intl` and `formatjs` take advantage of native internationalization features provided by web browsers.
+
+Language-specific content is located in YML files under the `i18n` folder of packages that have internationalizable content
+(e.g. `en-US.yml` for American English, `fr.yml` for generic French, etc.).
+
+### Contributing translations
+
+OTP-UI now uses [Weblate](https://www.weblate.org) to manage translations!
```

Comment:
```suggestion
OTP-UI now uses [Hosted Weblate](https://www.weblate.org) to manage translations!
<a href="https://hosted.weblate.org/engage/otp-react-redux/">
<img src="https://hosted.weblate.org/widgets/otp-react-redux/-/horizontal-auto.svg" alt="Translation status" />
</a>
```

Reply:
Thanks for the status images! Text and images added in https://github.com/opentripplanner/otp-ui/pull/465/commits/dc1b49a3c9dfc987e16c8bf64bae129bc903c563.